setup.py

```diff
@@ -54,26 +53,6 @@ def get_hash():
     return sha
```

Comment:
Looks like the `get_hash` and `get_git_hash` are no longer used?

Reply:
I think this func may be called by extra repo. ^_^